### PR TITLE
Add elut and sci channles for fine engery bins

### PR DESCRIPTION
**Pull request by samaloney**
* Add new lookup table for sci energy channels, add origin and new fine bins definitions
* Update ELUT tables and dates

### CODE REVIEW

**Review by nicHoch — APPROVED** [review]
looks good

File: elut/elut_index.csv [hunk 1]
```diff
@@ -4,6 +4,7 @@ index, start_date, end_date, elut_file
 2, 2021-05-25T14:55:49, 2021-12-09T00:00:00, elut_table_20210625.csv
 3, 2021-12-09T00:00:00, 2022-06-01T00:00:00, elut_table_20211209.csv
 4, 2022-06-01T00:00:00, 2022-09-18T00:00:00, elut_table_20220601.csv
-5, 2022-09-18T00:00:00, 2023-01-30T00:04:35, elut_table_20220918.csv
-6, 2023-01-30T00:04:35, 2023-02-02T16:06:12, elut_table_20221220.csv
-7, 2023-02-02T16:06:12, none, elut_table_20220918.csv
+5, 2022-09-18T00:00:00, 2023-01-19T17:02:40, elut_table_20220918.csv
+6  2023-01-19T17:02:40, 2023-01-30T00:04:35, elut_table_20221214.csv
```
Comment (grazwegian) [inline elut/elut_index.csv]: There is a missing comma after the index number

File: detector/ScienceEnergyChannels_01023023.csv [hunk 2]
```diff
@@ -0,0 +1,57 @@
+ENERGY BINNING,,,,,,,01.02.23
+,,,,,,,
+Energy binning with masks reset after January 2023 fine energy test but ELUT unchanged.,,,,,,,
+,,,,,,,
+Criteria,,,,,,,
+,Increasing bin width between 4 keV and 150 keV,,,,Include LLD and ULD channels < 4kev and > 150keV,,
+,"Fine (< 1kev) resolution through microflare regime, 4-11 keV",,,,"Bracket Ba-133 lines at 31, 35 and 81 keV",,
+,Bin centres at 6.7 and 8.0 keV (Fe and Fe/Ni line complexes),,,,,,
+,Sufficient coverage over full range,,,,,,
+,"Match current QL boundaries at 4, 10, 15, 25, 50, and 84 keV",,,,,,
```
Comment (grazwegian) [inline detector/ScienceEnergyChannels_01023023.csv]: Sorry, my mistake but this line shouldn't be here as this configuration does not match the standard QL boundaries.

Comment (samaloney) [reply detector/ScienceEnergyChannels_01023023.csv]: So I should just remove this `"Match current QL boundaries at 4, 10, 15, 25, 50, and 84 keV"`

Comment (grazwegian) [reply detector/ScienceEnergyChannels_01023023.csv]: [ScienceEnergyChannels_01022023.csv](https://github.com/i4Ds/STIX-CONF/files/10697389/ScienceEnergyChannels_01022023.csv)
Attached is an updated version with one of the additional criteria from column 6 moved to column 2. I also noticed there was a typo in the file name!

Comment (samaloney) [reply detector/ScienceEnergyChannels_01023023.csv]: What should that file be called, I guess it is the creation date the other one looks to be yyyymmdd,  `ScienceEnergyChannels_20221216.csv` and the new one ddmmyyyy `ScienceEnergyChannels_01023023.csv`? I mean it doesn't matter too much as the index file contains timings but just to be consistent.

**Pull request merged**